### PR TITLE
fdisk: add See also

### PR DESCRIPTION
**Pull request by aherst**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Have you seen #6169 @aherst ?

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/fdisk.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # fdisk
 
 > A program for managing partition tables and partitions on a hard disk.
+> See also: `partprobe`.
+> More information: <https://manned.org/fdisk>.
```
Comment (marchersimon) [inline pages/linux/fdisk.md:5]: Hey @aherst, the link for `fdisk` is already added in #6076
```suggestion
```

Comment (CleanMachine1) [reply pages/linux/fdisk.md:5]: Yes, this will stop a conflict between nothing. just makes future easier for others.

**Review by marchersimon — APPROVED** [review]
👍🏽

**Pull request merged**